### DISCUSSION TITLE
The difference between documentation

### DISCUSSION THREAD

**Discussion by vladelleus** [aws-ia/terraform-aws-eks-blueprints-addons#187] [Q&A] [answered]
Hi, can someone explain the difference between these docs and why in AWS EKS blueprints https://aws-ia.github.io/terraform-aws-eks-blueprints/v4.32.1/ there are a lot of add-ons that are not able to use inside of module? For example, in the main docs, there is a KEDA add-on, but in add-ons docs https://aws-ia.github.io/terraform-aws-eks-blueprints-addons/main/ , there is no such add-on.
![image](https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/assets/84621365/618f418f-ac9e-48ff-905f-ccdee2b9ffb3)
In the docs, it says, that I can use it:
![image](https://github.com/aws-ia/terraform-aws-eks-blueprints-addons/assets/84621365/c06104ff-b169-4946-9ff2-6a1075e3c412)

**Comment by bryantbiggs** [comment, marked as answer]
This has some of the context as to why https://github.com/aws-ia/terraform-aws-eks-blueprints/issues/1421

If you look at the main docs for EKS Blueprints, you will see the addons are no longer present https://aws-ia.github.io/terraform-aws-eks-blueprints/main/

And this goes further into the v5 changes https://aws-ia.github.io/terraform-aws-eks-blueprints/main/v4-to-v5/motivation/

**Reply by vladelleus** [reply]
Now I got all answers, thanks!
